cst/plugins/eu.esdihumboldt.cst.functions.numeric/src/eu/esdihumboldt/cst/functions/numeric/sequentialid/SequentialID.java

```diff
@@ -46,7 +44,7 @@
 	protected Object evaluate(String transformationIdentifier, TransformationEngine engine,
 			ListMultimap<String, PropertyValue> variables, String resultName,
 			PropertyEntityDefinition resultProperty, Map<String, String> executionParameters,
-			TransformationLog log) throws TransformationException, NoResultException {
+			TransformationLog log) throws RuntimeException {
```

Comment:
Any specific reason to change `throws TransformationException, NoResultException ` by ` throws RuntimeException`?

Reply:
I can go back to TransformationException, NoResultException and add InvalidInputException